content/ko/trademarks.md

```diff
@@ -0,0 +1,119 @@
+---
+title: 상표 정책
+slug: trademarks
+top_graphic: 1
+date: 2018-09-19
+lastmod: 2018-09-19
+---
```

Comment:
`english_is_canonical: 1` is missing: https://raw.githubusercontent.com/letsencrypt/website/master/content/en/trademarks.md